### ISSUE TITLE
Music Lounge: Doesn't work with keyboard

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#213]
Needs to work with keyboard for a11y reasons.

Perhaps each cube should be in the tab cycle and arrow keys (or asdw) could move the cubes. I assume Babylon.js has keyboard support.

**Comment by deltakosh** [comment]
Can you assign it to @davrous?

**Comment by dstorey** [comment]
@deltakosh it seems like it doesn't allow me to assign @davrous

**Comment by molant** [comment]
If he isn't part if the organization or the team he can't be added.

**Comment by sarvaje** [comment]
I've talked with @molant and @dstorey and I'll take care of this
